x-pack/plugins/security_solution/public/management/components/console/components/history_output.tsx

```diff
@@ -19,6 +21,16 @@ export const HistoryOutput = memo<OutputHistoryProps>((commonProps) => {
   const dispatch = useConsoleStateDispatch();
   const getTestId = useTestIdGenerator(useDataTestSubj());
 
+  const historyBody = useMemo(() => {
```

Comment:
Is there any advantage of abstracting this into a memo here insteaa of just mapping and rendering the items inside the `EuiFlexGroup` below?

Reply:
The only advantage I see is avoiding having to do the `.map()` on every render even if `historyItems` did not change. The console could have many items in the history. But then again, maybe this is a minor perf. improvement that is really not needed? (I'm not sure if this would re-render for other (not the history items) state changes 🤔